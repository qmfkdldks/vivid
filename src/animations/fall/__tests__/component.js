import React from "react";
import { shallow } from "enzyme";
import Fall from "../component";

describe("Fall", () => {
  it("should render successfully", () => {
    const component = shallow(<Fall>Text</Fall>);
    expect(component).toMatchSnapshot();
  });
});
