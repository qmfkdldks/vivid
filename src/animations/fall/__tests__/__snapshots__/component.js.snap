// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fall should render successfully 1`] = `
<Styled(Component)
  transition={
    Object {
      "duration": 5,
    }
  }
  variants={
    Object {
      "start": Object {
        "opacity": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
        ],
        "scaleX": Array [
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1.2,
          0.8,
          1.05,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
        ],
        "scaleY": Array [
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          0.7,
          1.2,
          0.95,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
        ],
        "y": Array [
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          -100,
          0,
          0,
          -15,
          0,
          -5,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
      },
      "stop": Object {
        "opacity": 1,
        "scaleX": 1,
        "scaleY": 1,
        "y": 0,
      },
    }
  }
>
  Text
</Styled(Component)>
`;
